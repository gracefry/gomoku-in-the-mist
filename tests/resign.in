who
place J1
who
resign
