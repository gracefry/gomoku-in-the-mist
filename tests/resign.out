B
W
Black wins!
J1
Thank you for playing!
